src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/CDMemberServiceImplementation.java

```diff
@@ -0,0 +1,48 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.CDMember;
+import ro.societateahermes.backendservice.entities.DTO.CDMemberDTO;
+import ro.societateahermes.backendservice.repository.CDMemberRepositoryInterface;
+import ro.societateahermes.backendservice.service.CDMemberServiceInterface;
+
+import java.util.List;
+
+@Service
+public class CDMemberServiceImplementation implements CDMemberServiceInterface {
+    private final CDMemberRepositoryInterface CDMemberRepository;
+
+
+    CDMemberServiceImplementation(CDMemberRepositoryInterface CDMemberRepo) {
```

Comment:
Foloseste @Autowired ba deasupra constructorului ba deasupra liniei in care declari campul